frontend/public/locales/en/data-unavailable.json

```
@@ -3,6 +3,15 @@
   "breadcrumbs": {
     "unable-to-continue": "Unable to continue"
   },
-  "service-unavailable": "This service is not available to you. We did not find any record for your account.",
+  "service-eligiblle": "You are currently not a participant in the Canadian Dental Care Plan, to find out if you are eligible please visit Do you qualify on Canada.ca",
+  "do-you-qualify.text": "Do you qualify - Canada.ca",
+  "do-you-qualify.href": "https://www.canada.ca/en/services/benefits/dental/dental-care-plan/qualify.html",
+  "more-information": "For more information on how to apply, please visit How to apply on Canada.c",
```

```suggestion
  "more-information": "For more information on how to apply, please visit How to apply on Canada.ca",
```
(same thing for French)

---

fixed